interfaces/Portalicious/src/app/services/registration-attribute.service.ts

```diff
@@ -213,7 +262,16 @@ export class RegistrationAttributeService {
             {
               name: 'name',
               label: $localize`:@@registration-full-name:Name`,
-              editInfo: $localize`:@@registration-full-name-edit-info:This field is dynamically generated based on the other name fields available below.`,
+              editInfo:
+                $localize`:@@registration-full-name-edit-info:This field is dynamically generated based on the other name fields available below: ` +
+                (project.fullnameNamingConvention
+                  ?.map((namingConvention) =>
+                    this.localizeAttribute({
+                      attributes: projectSpecificAttributes,
+                      attributeName: namingConvention,
+                    }),
+                  )
+                  .join(', ') ?? ''),
```

Comment:
```suggestion
                  .join($localize`:@@enumeration-separator:, `) ?? ''),
```

Possibly to prevent some potential "Right-to-Left"/Arabic translations issues...

Reply:
I found this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat

It looks like a really good solution, but then I would like to apply it systematically throughout the codebase rather than just here. What do you think? Shall I create a task for it?

Reply:
Ok created that here: https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/32521